CONTRIBUTING.rst

```diff
@@ -78,7 +78,7 @@ Development Workflow
          # Test your installation
          pytest --pyargs networkx
 
-   * Finally, we recommend you use a pre-commit hook, which runs black when
+   * Finally, we recommend you use a pre-commit hook, which runs ruff when
```

Comment:
Rather than call out the specific tools, it might be better to replace this with a more general statement, like "which ensures all code is linted and formatted correctly".

Reply:
Or maybe even more generally, "Finally, we recommend you use a pre-commit hook, which is useful to ensure your contribution will pass some of our CI."

That way if you decide to add more to the pre-commit-config.yaml you'll be less likely to need an update here again.

Reply:
> Or maybe even more generally, "Finally, we recommend you use a pre-commit hook, which is useful to ensure your contribution will pass some of our CI."

Sounds good!

Reply:
Ah wait, I think maybe just `will pass our CI`?

Reply:
I can modify it if you'd like, but I meant *"some"* since pre-commit typically isn't used to cover *all* of CI (i.e. tests).

Reply:
At the risk of bike-shedding, I'd actually advocate for the original wording I proposed. The reasons being:
 1. Some contributors may not know what CI is, and those that do may not have the full context to understand the statement. I wouldn't want to send folks down rabbit holes or make them unsure.
 2. `pre-commit` is not only used as a git hook, but is also the primary interface to the formatting and linting for the project and is used outside of the commit-hook context. For example, see step 4 of the development workflow, where contributors can run linting/formatting on the entire codebase with `pre-commit run --all-files`.

Reply:
Okay. I guess just let me know what the consensus is and I'll update it.